pykern/pkunit.py

```diff
@@ -83,6 +83,11 @@ def case_dirs(group_prefix=''):
     of these expect files is copmared to the corresponding `work_dir`
     actual file using `file_eq`.
 
+    If you want to only use cases from some specific `/<case-name>.in`
+    subdir, and not all `/*.in` subdirs, you can pass a `group_prefix`
+    default parameter value ('' by default) to `case_dirs()`. This will
+    perform the regular operations but only on `/<case-name>.in`.
+
```

Comment:
Add an `Args:` section. See example in [pkjson](https://github.com/radiasoft/pykern/blob/3c0eda21925a2df323299b6658d5ac5c216335a0/pykern/pkjson.py#L29) for how to handle optional args.

Reply:
Still don't see this.

Reply:
It's in most recent commit